script.js

```diff
@@ -0,0 +1,11 @@
+const menuIcon = document.getElementById('menu-button');
+const close = document.getElementById('Close-btn');
+const menu = document.getElementById('menu-container');
+
+menuIcon.addEventListener('click', function () {
+  menu.classList.toggle('hidden');
+})
+
+close.addEventListener('click', function () {
+  menu.classList.toggle('hidden');
```

Comment:
- [ ] When a user clicks one of the links (options) in the mobile menu, a correct part of the page should be displayed and the mobile menu should disappear. Your mobile menu doesn't function this way. Kindly work on it. 👍